renderer.php

```diff
@@ -24,6 +25,13 @@ public function __construct() {
 		$this->m_exclude =
 			(!empty($inclusions) && !preg_match("/$inclusions/", $ID)) ||
 			(!empty($exclusions) && preg_match("/$exclusions/", $ID));
+
+		// Set the regex for filtering link destinations
+		$points = $this->getConf('linkall_points_to');
+		if (empty($points)) {
+			$points = ".*";
+		}
```

Comment:
It would be more efficient to leave this empty, and test `empty()` in `internallink`, rather than force an extra regex test every time.